docs/guides/distribution/updater.md

```diff
@@ -2,220 +2,126 @@
-
-Add this in tauri.conf.json
-
-```json
-"updater": {
-    "active": true,
-    "endpoints": [
-        "https://releases.myapp.com/{{target}}/{{current_version}}"
-    ],
-    "dialog": true,
-    "pubkey": "YOUR_UPDATER_SIGNATURE_PUBKEY_HERE"
-}
-```
-
-The required keys are "active", "endpoints" and "pubkey"; others are optional.
-
-"active" must be a boolean. By default, it's set to false.
+import Command from '@theme/Command'
 
-"endpoints" must be an array. The string `{{target}}` and `{{current_version}}` are automatically replaced in the URL allowing you to determine [server-side](#update-server-json-format) if an update is available. If multiple endpoints are specified, the updater will fallback if a server is not responding within the pre-defined timeout.
-
-"dialog" if present must be a boolean. By default, it's set to true. If enabled, [events](#events) are turned off as the updater handles everything. If you need the custom events, you MUST turn off the built-in dialog.
-
-"pubkey" must be a valid public-key generated with Tauri CLI. See [Signing updates](#signing-updates).
+# Updater
 
-### Update Requests
+Tauri offers a built-in updater for the MSI (Windows), AppImage (Linux) and App bundle (macOS) distribution formats.
 
-Tauri is indifferent to the request the client application provides for update checking.
+## Configuration
 
-`Accept: application/json` is added to the request headers because Tauri is responsible for parsing the response.
+Once your Tauri project is ready, you can configure Tauri's updater to enable auto updating for your users.
 
-For the requirements imposed on the responses and the body format of an update, response see [Server Support](#server-support).
+### Signing updates
 
-Your update request must _at least_ include a version identifier so that the server can determine whether an update for this specific version is required.
+Tauri's updater has a built-in signature to ensure updates are safe to be installed. This safety measurement can not be turned off.
 
-It may also include other identifying criteria, such as operating system version, to allow the server to deliver as fine-grained an update as you would like.
+To sign your updates you need two things:
 
-How you include the version identifier, or other criteria is specific to the server from which you request updates. A common approach is to use query parameters, [Configuration](#configuration) shows an example.
+- The _public key_, which will be added to your `tauri.conf.json` file later, to validate update artifacts before the installation.
+- The _private key_, which is used to sign your update artifacts and should NEVER be shared with anyone. Also, if you loose this key, you will NOT be able to publish new updates to your current user base. It is crucial to store it in a safe place where you can always access it.
 
-### Built-in dialog
+To generate the keys, you need to use the Tauri CLI (use `$HOME/.tauri/myapp.key` if you're on Windows):
```

Comment:
I think the `use <blah> on Windows` should be moved under the command not before it.